examples/sara_work_area/genz_51_list_reversal

```diff
@@ -0,0 +1,3 @@
+test_list = [1,2,3,4]
```

Comment:
According to PEP-8 coding standards, we should have a space after comma ,
so test_list will look like [1, 2, 3, 4]